treeherder/model/management/commands/cycle_data.py

```diff
@@ -10,6 +10,16 @@
                                      Repository)
 from treeherder.perf.models import PerformanceDatum
 
+env = environ.Env()
+
+# Default to retaining data for ~4 months.
+DATA_CYCLE_DAYS = env.int("DATA_CYCLE_DAYS", default=120)
+
+# Determines the number of jobs we try to delete per iteration when
+# cycling data
+DATA_CYCLE_CHUNK_SIZE = env.int("DATA_CYCLE_CHUNK_SIZE", default=100)
+DATA_CYCLE_SLEEP_TIME = env.int("DATA_CYCLE_SLEEP_TIME", default=0)
```

Comment:
We haven't used any of these `DATA_CYCLE_*` settings for a while (previously different expiration times were set per environment), so let's just remove the options and transpose the defaults (and comments) below.